### PR TITLE
Feature/619 capacity unit

### PR DESCRIPTION
**Pull request by ismaelbej**

### CODE REVIEW

**Review by ggiraldez — APPROVED** [review]
Looks good, although it includes all the changes in #631. All comments are suggestions to improve the codebase as usual.

File: client/src/planwise/client/projects2/components/dashboard.cljs [hunk 1]
```diff
@@ -11,7 +11,7 @@
             [planwise.client.ui.rmwc :as m]
             [planwise.client.utils :as utils]
             [planwise.client.projects2.components.settings :as settings]
-            [planwise.common :as common]))
+            [planwise.common :refer [get-consumer-unit get-demand-unit get-provider-unit get-capacity-unit] :as common]))
```
[the remark this answers is not in the document]
Comment (ismaelbej) [reply client/src/planwise/client/projects2/components/dashboard.cljs:14]: Sorry! My mistake. It is based on #631. The PR should be fixed now.

File: client/src/planwise/client/scenarios/edit.cljs [hunk 2]
```diff
@@ -122,7 +122,7 @@
 
 (defn- action->title
   [name]
-  (str/join " " (map str/capitalize (str/split name #"-"))))
+  (str/capitalize (first (str/split name #"-"))))
```
[the remark this answers is not in the document]
Comment (ismaelbej) [reply client/src/planwise/client/scenarios/edit.cljs]: I'll do it.

File: client/src/planwise/client/utils.cljs [hunk 3]
```diff
@@ -65,9 +65,13 @@
          format-string (str "%." decimals "f%%")]
      (gstring/format format-string percentage))))
 
+(defn format-value
```
[the remark this answers is not in the document]
Comment (ismaelbej) [reply client/src/planwise/client/utils.cljs:68]: It seems currency is the term more commonly used so I'll go with format-currency.

File: client/src/planwise/client/utils.cljs [hunk 4]
```diff
@@ -65,9 +65,13 @@
...
+  [value]
+  (str common/currency-symbol " " (format-number value)))
+
 (defn format-effort
   [effort analysis-type]
-  (str (if (common/is-budget analysis-type) (str common/currency-symbol " ")) (format-number effort)))
+  (str ((if (common/is-budget analysis-type) format-value format-number) effort)))
```
[the remark this answers is not in the document]
Comment (ismaelbej) [reply client/src/planwise/client/utils.cljs:74]: I'll do it.

**Pull request merged**